docs/source/conf.py

```diff
@@ -381,4 +385,13 @@ def reformat_md_section_links(file_path: Path):
     "gallery_dirs": ["auto_examples", "substrafl_doc/examples"],
     "subsection_order": SubSectionTitleOrder("../../examples"),
     "download_all_examples": False,
+    'binder': {
+        'org': 'Substra',
+        'repo': 'substra-documentation',
+        'branch': current_commit, # Can be any branch, tag, or commit hash. Use a branch that hosts your docs.
+        'binderhub_url': 'https://mybinder.org', # Any URL of a binderhub deployment. Must be full URL (e.g. https://mybinder.org).
```

Comment:
what is the value we should put here?

Reply:
mybinder.org, it is the public instance of binder

Reply:
ok, we can delete the comment in that case